packages/web-client/app/components/card-pay/create-merchant-workflow/next-steps/index.ts

```diff
@@ -8,7 +8,9 @@ class CardPayCreateMerchantWorkflowNextStepsComponent extends Component<Workflow
   @service declare router: RouterService;
 
   @action returnToDashboard() {
-    this.router.transitionTo({ queryParams: { flow: null } });
+    this.router.transitionTo({
+      queryParams: { flow: null, worklowPersistenceId: null },
```

Comment:
```suggestion
      queryParams: { flow: null, workFlowPersistenceId: null },
```

Reply:
Had to change this to `'flow-id': null`  instead of `workflowPersistenceId: null ` for both workflows. Otherwise clicking on that button gave an error and told me to use 'flow-id'.